themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,32 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'div';
+  $wrapperClass = $this->wrapperClass ?? false;
+
+  $toggleTag = $this->toggleTag ?? 'button';
+  $toggleAttrs = $this->toggleAttrs ?? [];
+
+  $href = $this->href ?? $toggleTag == 'button' ? null : '#';
+?>
+
+<<?=$wrapperTag ?>
+  class="<?=$wrapperClass ? $this->escapeHtmlAttr($wrapperClass) . ' ' : '' ?>dropdown"
+>
+  <<?=$toggleTag ?>
```

Comment:
Would this stuff be more readable if we used the makeTag view helper, now that we have it? Or do we want to avoid that in order to keep templates more "pure PHP/HTML"? I know there's been some back-and-forth debate about strategy here. I personally think the helper would actually help here, but maybe I'm wrong. :-)